openlibrary/i18n/__init__.py

```diff
@@ -12,6 +12,8 @@
 
 root = os.path.dirname(__file__)
 
+OL_SUPPORTED_LANGAGES = ['en', 'cs', 'de', 'es', 'fr', 'te']
```

Comment:
This is going to fall out of sync ; in fact it already is! It's missing `hr`. Let's update and memoize the `get_locales` function to checks the children directories of this directory that contain `messages.po`. Memoize similar to `load_translations`. Might be able to use `glob`? Not sure how to make it use this file's directory (`root` above).

```py
>>> glob.glob('*/*messages.po')
['cs\\messages.po', 'de\\messages.po', 'es\\messages.po', 'fr\\messages.po', 'hr\\messages.po', 'te\\messages.po']
```